nautobot/extras/models/jobs.py

```diff
@@ -594,6 +593,8 @@ class JobResult(BaseModel, CustomFieldModel):
 
     objects = JobResultManager()
 
+    _natural_key = ["task_id"]
```

Comment:
Needed presently because `task_id` is `unique=True` but since it's a `UUIDField` it gets overlooked by django-natural-keys.